tests/e2e_tests/test_storage_compatibility.py

```
@@ -0,0 +1,207 @@
+import uuid
+
+import pytest
+import requests
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)